esp/services/ws_sql/SQL2ECL/HPCCSQLTreeWalker.cpp

```diff
@@ -324,6 +326,14 @@ ISQLExpression * HPCCSQLTreeWalker::expressionTreeWalker(pANTLR3_BASE_TREE exprA
                         paramList.append(*rightexp.getLink());
                     }
                 }
+                if (exptype == LIKE_SYM || exptype == NOT_LIKE)
+                {
+                    if (rightexp->getExpType() == Value_ExpressionType)
+                    {
+                        SQLValueExpression * valexp = static_cast<SQLValueExpression *>(rightexp.get());
+                        valexp->setIsWildCardPattern(true);
+                    }
```

Comment:
Good question, in general an if should also account for the else. In this case however, the else would have already been taken care of by lines 312-328...
I could have not processed case LIKE_SYM and case NOT_LIKE on their own, but I didn't want to duplicate 312-315 in the new case...